docs/modules/ROOT/pages/how-tos/exoscale/install.adoc

```diff
@@ -70,23 +134,129 @@ EOF
+  -e 's/openstack/exoscale/'
+
+exo storage upload rhcos-${RHCOS_VERSION}.qcow2 "sos://${CLUSTER_ID}-bootstrap" --acl public-read
+
+exo vm template register "rhcos-${RHCOS_VERSION}" \
+  --checksum $(md5sum rhcos-${RHCOS_VERSION}.qcow2 | awk '{ print $1 }') \
+  --boot-mode uefi \
+  --disable-password \
+  --username core \
+  --description "Red Hat Enterprise Linux CoreOS (RHCOS) ${RHCOS_VERSION}" \
+  --url "https://${AWS_S3_ENDPOINT}/${CLUSTER_ID}-bootstrap/rhcos-${RHCOS_VERSION}.qcow2"
+
+exo storage delete "sos://${CLUSTER_ID}-bootstrap/rhcos-${RHCOS_VERSION}.qcow2"
+
+export RHCOS_TEMPLATE="rhcos-${RHCOS_VERSION}"
+----
+
+. Upload Ursula Debian package to bootstrap bucket
++
+[source,console]
+----
+exo storage upload /path/to/ursula.deb "sos://${CLUSTER_ID}-bootstrap/ursula.deb" --acl public-read
+----
+
+
+=== Set secrets in Vault
+
+.Connect with Vault
+[source,console]
+----
+export VAULT_ADDR=https://vault-prod.syn.vshn.net
+vault login -method=ldap username=<your.name>
+----
+
+.Store various secrets in Vault
+[source,console]
+----
+# Set the exoscale.ch S3 secret
+vault kv put clusters/kv/${TENANT_ID}/${CLUSTER_ID}/exoscale \
+  s3_access_key=${AWS_ACCESS_KEY_ID} \
+  s3_secret_key=${AWS_SECRET_ACCESS_KEY}
+
+# Generate an HTTP secret for the registry
+vault kv put clusters/kv/${TENANT_ID}/${CLUSTER_ID}/registry \
+  httpSecret=$(LC_ALL=C tr -cd "A-Za-z0-9" </dev/urandom | head -c 128)
+
+# Set the LDAP password
+vault kv put clusters/kv/${TENANT_ID}/${CLUSTER_ID}/vshn-ldap \
+  bindPassword=${LDAP_PASSWORD}
+
+# Generate a master password for K8up backups
+vault kv put clusters/kv/${TENANT_ID}/${CLUSTER_ID}/global-backup \
+  password=$(LC_ALL=C tr -cd "A-Za-z0-9" </dev/urandom | head -c 32)
+
+# Generate a password for the cluster object backups
+vault kv put clusters/kv/${TENANT_ID}/${CLUSTER_ID}/cluster-backup \
+  password=$(LC_ALL=C tr -cd "A-Za-z0-9" </dev/urandom | head -c 32)
+
+# Copy the Dagobert OpenShift Node Collector Credentials
+vault kv get -format=json "clusters/kv/template/dagobert" | jq '.data.data' \
+  | vault kv put -cas=0 "clusters/kv/${TENANT_ID}/${CLUSTER_ID}/dagobert" -
+----
+
+=== OpenShift Installer Setup
+
+For the following steps, change into a clean directory (for example a directory in your home).
+
+[CAUTION]
+These are the only steps which aren't idempotent and have to be completed uninterrupted in one go.
+If you have to recreate the install config or any of the generated manifests you need to rerun all of the subsequent steps.
+
+[NOTE]
+--
+You can add more options to the `install-config.yaml` file.
+Have a look at the https://docs.openshift.com/container-platform/{ocp-minor-version}/installing/installing_bare_metal/installing-bare-metal.html#installation-bare-metal-config-yaml_installing-bare-metal[config example] for more information.
+
+For example, you could change the SDN from a default value to something a customer requests due to some network requirements.
+--
+
+. Prepare SSH key
```

Comment:
```suggestion
. Prepare SSH key
+
NOTE: We generate a unique SSH key pair for the cluster, because Exoscale doesn't support configuring multiple public keys for a VM.
```